src/text/font.d

```diff
@@ -37,10 +37,10 @@ public void registerDefaultFonts () {
         FontRegistry.registerFont("georgia-italic", "/Library/Fonts/Georgia Italic.ttf");
         FontRegistry.registerFont("georgia-bold-italic", "/Library/Fonts/Georgia Bold Italic.ttf");
     }
-    version(Linux) {
+    version(linux) {
         // FontRegistry.registerFont(<string id>, <path to .ttf or .ttc file>, <optional index iff .ttc w/ multiple fonts>)
-        FontRegistry.registerFont("menlo", "your-font-file-here");
-        FontRegistry.registerFont("arial", "your-font-file-here");
+        FontRegistry.registerFont("menlo", "/usr/share/fonts/truetype/droid/DroidSansMono.ttf");
+        FontRegistry.registerFont("arial", "/usr/share/fonts/truetype/droid/DroidSans.ttf");
```

Comment:
For more generality we should probably change "menlo" and "arial" strings to be "fixed-width" and "sans".  Then we could define some globals FIXED_WIDTH_FONT and SANS_FONT somewhere and then use those instead of creating local FONT variables in multiple places throughout the code.